src/System.Runtime/tests/System/Array.cs

```diff
@@ -1243,6 +1295,16 @@ private static bool ArraysAreEqual<T>(T[] a, T[] b, IComparer<T> comparer)
     }
 
     [Fact]
+    public static void TestSort_Invalid()
+    {
+        IComparer<int> icomparer = new IntegerComparer();
+        Assert.Throws<ArgumentNullException>(() => Array.Sort(null, 0, 0, icomparer));
+        Assert.Throws<ArgumentOutOfRangeException>(() => Array.Sort(new int[] { 1 }, -1, 1, icomparer));
+        Assert.Throws<ArgumentOutOfRangeException>(() => Array.Sort(new int[] { 1 }, 1, -1, icomparer));
+        Assert.Throws<ArgumentException>(() => Array.Sort(new int[] { 1 }, 1, 1, icomparer));
```

Comment:
Here and above in the other invalid methods, we seem to only be calling the overloads with the most parameters.  Should there be similar "invalid" tests for all overloads?